src/components/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useCallback } from 'react';
+
+import MobileStepperMaterial from '@mui/material/MobileStepper';
+import classNames from 'classnames/bind';
+
+import { Button } from '@components/Button';
+import { SvgIcon } from '@components/SvgIcon';
+import { SwipeableViews } from '@components/SwipeableViews';
+import { Typography } from '@components/Typography';
+
+import styles from './Carousel.module.css';
+
+const cn = classNames.bind(styles);
+
+type CarouselContent = { id: string; imagePath: string }[];
+
+export type CarouselProps = {
+  carouselContent: CarouselContent;
+  title: string;
+  className?: string;
+  imgClassName?: string;
+  stepperClassName?: string;
+};
+
+export function Carousel({ carouselContent, title, imgClassName, className, stepperClassName }: CarouselProps) {
+  const [activeStep, setActiveStep] = useState(0);
+  const maxSteps = carouselContent.length;
+
+  const handleNext = useCallback(() => {
+    setActiveStep((prevActiveStep) => prevActiveStep + 1);
+  }, [activeStep]);
+
+  const handleBack = useCallback(() => {
+    setActiveStep((prevActiveStep) => prevActiveStep - 1);
+  }, [activeStep]);
+
+  const handleStepChange = useCallback((step: number) => {
+    setActiveStep(step);
+  }, []);
+
+  return (
+    <div className={cn('carousel', className)}>
+      <div className={cn('carousel__header')}>
+        <Typography className={cn('carousel__header-heading')}>{title}</Typography>
+        <div>
+          <Button
+            className={cn('carousel__button')}
+            disabled={activeStep === 0}
+            noHover
+            onClick={handleBack}
+            disableRipple
+          >
+            <SvgIcon className={cn('carousel__icon')} icon="circleChevronLeft" />
+          </Button>
+          <Button
+            className={cn('carousel__button')}
+            disabled={activeStep === maxSteps - 1}
+            noHover
+            onClick={handleNext}
+            disableRipple
+          >
+            <SvgIcon className={cn('carousel__icon')} icon="circleChevronRight" />
+          </Button>
+        </div>
+      </div>
+
+      <SwipeableViews index={activeStep} onChangeIndex={handleStepChange}>
+        {carouselContent.map((step, index) => (
+          <div key={step.id}>
+            {Math.abs(activeStep - index) <= 2 ? (
```

Comment:
nit: I would format ternary a little bit differently with '?' and ':' on new string.

Couldn't understand the meaning of this check

Reply:
the prettier auto formats based on formatting rules can't help it (